broker-core/src/main/java/io/zeebe/broker/logstreams/processor/StreamProcessorIds.java

```diff
@@ -17,32 +17,6 @@
  */
 package io.zeebe.broker.logstreams.processor;
 
-import io.zeebe.util.buffer.BufferUtil;
-import org.agrona.DirectBuffer;
-
 public class StreamProcessorIds {
-  // a stream processor partitionId should be unique to distinguish event producers
-
-  public static final int JOB_STREAM_PROCESSOR_ID = 10;
-
-  public static final int DISTRIBUTE_PROCESSOR_ID = 60;
-
-  public static final int WORKFLOW_INSTANCE_PROCESSOR_ID = 70;
-
-  public static final int INCIDENT_PROCESSOR_ID = 80;
-
-  public static final int MESSAGE_PROCESSOR_ID = 90;
-
   public static final int EXPORTER_PROCESSOR_ID = 1003;
```

Comment:
If this is the only constant in this file, let's move it to the exporter stream processor, no?

Reply:
aggree